tests/plugins/mongo/test_get_sv.py

```diff
@@ -1,12 +1,15 @@
 from loqusdb.build_models.variant import build_variant
+from loqusdb.constants import GRCH37
```

Comment:
Lots of tests with 37 (makes senses as it was and is default) but would be good with a couple on 38, in particular for the new order constant.